Procfile

```diff
@@ -0,0 +1 @@
+web: bundle exec rails server -p 5000
```

Comment:
this is a defunct way of running things. left over from when we used to run on heroku.